pennylane/qchem/givens_decomposition.py

```diff
@@ -86,7 +85,7 @@ def givens_decomposition(unitary):
 
     .. code-block:: python
 
-        unitary = np.array([[ 0.73678+0.27511j, -0.5095 +0.10704j, -0.06847+0.32515j],
+        unitary = qml.math.array([[ 0.73678+0.27511j, -0.5095 +0.10704j, -0.06847+0.32515j],
```

Comment:
We've not really advertised `qml.math.array` as a function for users to instantiate an array/tensor. Instead, users can pick their interface (numpy, torch, tf, jax) and `qml.math` will work with all of them.

So here, `np.array` seems fine to me :+1:

Reply:
Ok!